### PR TITLE
Feat/manager 7390: onboarding page

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/anthos`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-7390
| License          | BSD 3-Clause

## Description

Onboarding page

### CODE REVIEW

File: packages/manager/modules/anthos/src/onboarding/template.html [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+        <span data-translate="anthos_onboarding_description_part_1"></span>
+        <span
+            data-translate="anthos_onboarding_description_part_2"
+            class="bold"
+        ></span>
+        <span data-translate="anthos_onboarding_description_part_3"></span>
+    </p>
+
+    <!--Start with Anthos CTA-->
+    <a data-ng-href="{{$ctrl.anthosReadMoreInfoLink}}">
```
Comment (varun257) [inline packages/manager/modules/anthos/src/onboarding/template.html:15]:
```suggestion
    <a data-ng-href="{{:: $ctrl.anthosReadMoreInfoLink}}">
```

File: packages/manager/modules/anthos/src/onboarding/template.html [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
-    <oui-button data-variant="primary" data-on-click="$ctrl.discover()">
-        <span>Discover product</span>
-    </oui-button>
+<manager-on-boarding-layout data-image-source="$ctrl.illustration"
+                            data-guides="$ctrl.guides">
+    <!--Title and description-->
+    <h1 data-translate="anthos_onboarding_title"></h1>
+    <p class="mb-3">
+        <span data-translate="anthos_onboarding_description_part_1"></span>
+        <span
```
Comment (varun257) [inline packages/manager/modules/anthos/src/onboarding/template.html:7]: Can use `<strong>` instead

File: packages/manager/modules/anthos/src/onboarding/routing.js [hunk 3]
```diff
@@ -1,20 +1,48 @@
...
             ? {
                 state: 'anthos.index',
               }
-            : false,
-        ),
+            : false;
+        }),
     resolve: {
-      breadcrumb: () => null,
+      breadcrumb: ($translate) =>
```
Comment (varun257) [inline packages/manager/modules/anthos/src/onboarding/routing.js:20]:
```suggestion
breadcrumb: /* @ngInject */ ($translate) =>
```

File: packages/manager/modules/anthos/src/anthos.routing.js [hunk 4]
```diff
@@ -8,13 +8,22 @@ export default /* @ngInject */ ($stateProvider) => {
...
-              }
-            : false,
-        ),
+        .then((tenants) => {
+          if (tenants.length === 0) {
+            return {
+              state: 'anthos.onboarding',
+            };
+          }
+          if (tenants.length === 1) {
```
Comment (varun257) [inline packages/manager/modules/anthos/src/anthos.routing.js:17]: It should be an `else` or a fall through here, no need to check for length twice. If you are comfortable go with `tenants.length > 0`

File: packages/manager/modules/anthos/src/onboarding/template.html (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/onboarding/template.html:15]: Done

File: packages/manager/modules/anthos/src/onboarding/template.html (hunk 2 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/onboarding/template.html:7]: done

File: packages/manager/modules/anthos/src/onboarding/routing.js (hunk 3 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/onboarding/routing.js:20]: Nice, done

File: packages/manager/modules/anthos/src/anthos.routing.js (hunk 4 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/anthos.routing.js:17]: I am not sure to get your point of view.
The idea is :
customer target list page if we have 
   0 tenants => go to onboarding
   1 tenants => go to details page
   > 1 => can display list of tenants

Comment (mohammed-zahaf) [reply packages/manager/modules/anthos/src/anthos.routing.js:17]: @varun257 let me know if you need more explantion

File: packages/manager/modules/anthos/src/onboarding/routing.js [hunk 5]
```diff
@@ -1,20 +1,48 @@
...
+      illustration: /* @ngInject */ () => illustration,
+
+      guides: /* @ngInject */ ($translate, ovhSubsidiary) => {
+        return [
+          {
+            title: $translate.instant('anthos_onboarding_guides_anthos_title'),
+            description: $translate.instant(
+              'anthos_onboarding_guides_anthos_description',
+            ),
+            link: GUIDES[ovhSubsidiary] || GUIDES.DEFAULT,
```
Comment (radireddy) [inline packages/manager/modules/anthos/src/onboarding/routing.js:35]: 👍

**Review by radireddy — APPROVED** [review]

**Review by varun257 — APPROVED** [review]

**Pull request merged**